<ui version="4.0" >
 <class>DlgEditorBase</class>
 <widget class="QWidget" name="DlgEditorBase" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>375</width>
    <height>108</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3" >
   <property name="margin" >
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox_2" >
     <property name="title" >
      <string>Editor</string>
     </property>
     <layout class="QGridLayout" name="gridLayout" >
      <item row="0" column="0" >
       <widget class="QLabel" name="label" >
        <property name="text" >
         <string>Editor:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QComboBox" name="kcfg_ExternalEditor" >
        <property name="whatsThis" >
         <string comment="@info:whatsthis" >Choose the editor you want to launch when Okular wants to open a source file.</string>
        </property>
       </widget>
      </item>
      <item rowspan="2" row="1" column="1" >
       <widget class="QStackedWidget" name="stackCommands" >
        <property name="currentIndex" >
         <number>1</number>
        </property>
        <widget class="QWidget" name="page" >
         <layout class="QVBoxLayout" name="verticalLayout_2" >
          <property name="margin" >
           <number>0</number>
          </property>
          <item>
           <widget class="QLineEdit" name="kcfg_ExternalEditorCommand" />
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="page_2" >
         <layout class="QVBoxLayout" name="verticalLayout" >
          <property name="margin" >
           <number>0</number>
          </property>
          <item>
           <widget class="QLineEdit" name="leReadOnlyCommand" >
            <property name="readOnly" >
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item row="2" column="0" >
       <widget class="QLabel" name="label_2" >
        <property name="text" >
         <string>Command:</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>20</width>
       <height>4</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
